.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: cpp
+
+notifications:
+  email: false
+
+sudo: required
+dist: trusty
+
+branches:
+  only:
+    - master
+    - B3_0_Release
```

Comment:
@mariuz B2_5_Release is not listed here.

Did you tested the builds commands in Ubuntu trusty? I do not remember about this specific version, but I always have trouble building 2.5 in newer Ubuntu versions.

Reply:
On trusty firebird 2.5 branch builds ok 

On 17.10 i have an error (segfault when create_db empty.fdb)